pkg/loop/internal/pb/relayer.proto

```diff
@@ -0,0 +1,262 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-relay/pkg/loop";
+
+package loop;
+
+import "google/protobuf/empty.proto";
+
+service PluginRelayer {
+  rpc NewRelayer (NewRelayerRequest) returns (NewRelayerReply) {}
+}
+
+message NewRelayerRequest {
+  string config = 1; // toml (is chain instance config enough?)
+  uint32 keystoreID = 2;
+
+  //TODO prometheus? https://smartcontract-it.atlassian.net/browse/BCF-2075
+}
+
+message NewRelayerReply {
+  uint32 relayerID = 1;
+}
+
+service Keystore {
+  rpc Keys (google.protobuf.Empty) returns (KeysReply) {}
+  rpc Sign (SignRequest) returns (SignReply) {}
+}
+
+message KeysReply {
+  repeated string accounts = 1;
+}
+
+message SignRequest {
+  string account = 1;
+  bytes data = 2;
+}
+
+message SignReply {
+  bytes signed = 1;
+}
+
+service Relayer {
+  rpc NewConfigProvider (NewConfigProviderRequest) returns (NewConfigProviderReply) {}
+  rpc NewMedianProvider (NewMedianProviderRequest) returns (NewMedianProviderReply) {}
+  rpc NewMercuryProvider (NewMercuryProviderRequest) returns (NewMercuryProviderReply) {}
+
+  rpc ChainStatus (ChainStatusRequest) returns (ChainStatusReply) {}
+  rpc ChainStatuses (ChainStatusesRequest) returns (ChainStatusesReply) {}
+
+  rpc NodeStatuses (NodeStatusesRequest) returns (NodeStatusesReply) {}
+
+  rpc SendTx (SendTxRequest) returns (google.protobuf.Empty) {}
+}
+
+message RelayArgs {
+  bytes externalJobID = 1; // [32]byte
+  int32 jobID = 2;
+  string contractID = 3;
+  bool new = 4;
+  bytes relayConfig = 5;
+}
+
+message PluginArgs {
+  string transmitterID = 1;
+  bytes pluginConfig = 2;
```

Comment:
Just matching this:
https://github.com/smartcontractkit/chainlink-relay/blob/main/pkg/types/types.go#L27
Nothing imposed by this API since they are defined per plugin (as in OCR2 `ReportingPlugin` - not LOOPP).